### ISSUE TITLE
Unable to enter command mode in setup cell

### ISSUE THREAD

**Issue by manzt** [marimo-team/marimo#6524]
### Describe the bug

From Discord: https://discord.com/channels/1059888774789730424/1420553149323677779

Cmd + Esc does not work within setup cell on macOS to enter command mode. I believe I've run into something similar before where keyboard navigation doesn't work in command mode for setup cells.

https://github.com/user-attachments/assets/0c1aba3a-ef57-4ada-a0b9-d1ad6937492e

### Will you submit a PR?

- [ ] Yes

### Environment

```json
{
  "marimo": "0.15.5",
  "editable": false,
  "location": "/Users/manzt/.local/share/uv/tools/marimo/lib/python3.13/site-packages/marimo",
  "OS": "Darwin",
  "OS Version": "24.6.0",
  "Processor": "arm",
  "Python Version": "3.13.3",
  "Binaries": {
    "Browser": "140.0.7339.187",
    "Node": "v24.4.1"
  },
  "Dependencies": {
    "click": "8.2.1",
    "docutils": "0.21.2",
    "itsdangerous": "2.2.0",
    "jedi": "0.19.2",
    "markdown": "3.8.2",
    "narwhals": "1.44.0",
    "packaging": "25.0",
    "psutil": "7.0.0",
    "pygments": "2.19.2",
    "pymdown-extensions": "10.16",
    "pyyaml": "6.0.2",
    "starlette": "0.47.1",
    "tomlkit": "0.13.3",
    "typing-extensions": "missing",
    "uvicorn": "0.34.3",
    "websockets": "15.0.1"
  },
  "Optional Dependencies": {
    "loro": "1.5.2"
  },
  "Experimental Flags": {
    "performant_table_charts": true,
    "sql_linter": true,
    "multi_column": true,
    "mcp_docs": true
  }
}
```

**Comment by mscolnick** [comment]
Fixed in https://github.com/marimo-team/marimo/releases/tag/0.16.5
